src/store/lookup.ts

```diff
@@ -117,7 +117,10 @@ async function lookupCard(
 	}
 
 	if (await lookupCardInStock(store, page, link)) {
-		const givenUrl = link.cartUrl ? link.cartUrl : link.url;
+		const givenUrl =
+			link.cartUrl && !envOrBoolean(process.env.DISABLE_ADD_TO_CART, false)
```

Comment:
I still think we should put this in config and export that rather than the `envOrBoolean`.

Reply:
Agreed, I also switch the env var to be a positive rather than a negative so I didn't have to use a ! in the ternary.  Reads cleaner in my book.